plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtGetUnmanagedInstancesCommandWrapper.java

```diff
@@ -174,6 +174,9 @@ private List<UnmanagedInstanceTO.Nic> getUnmanagedInstanceNics(List<LibvirtVMDef
             nic.setNetwork(interfaceDef.getDevName());
             nic.setPciSlot(interfaceDef.getSlot().toString());
             nic.setVlan(interfaceDef.getVlanTag());
+            if (nic.getVlan() == -1 && interfaceDef.getNetType() == LibvirtVMDef.InterfaceDef.GuestNetType.BRIDGE) {
```

Comment:
```suggestion
            if (nic.getVlan() == -1
                && LibvirtVMDef.InterfaceDef.GuestNetType.BRIDGE.equals(interfaceDef.getNetType())) {
```

Reply:
@abhisar, in context this may be a non-issue but we can not guarantee the list will never contain a null, so as a matter of defensive programming.